contributing.md

```diff
@@ -0,0 +1,100 @@
+Contributing
+============
+
+All projects under the Pylons Projects, including this one, follow the
+guidelines established at [How to
+Contribute](http://www.pylonsproject.org/community/how-to-contribute) and
+[Coding Style and
+Standards](http://docs.pylonsproject.org/en/latest/community/codestyle.html).
+
+You can contribute to this project in several ways.
+
+* [File an Issue on GitHub](https://github.com/Pylons/waitress/issues)
+* Fork this project and create a branch with your suggested change. When ready,
```

Comment:
Generally I would prefer discussion related to the change the user would like to make before showing up with a pull request unless it is something minor.

The reason for this is that some changes can have wide ranging implications and may need to be discussed on how to best tackle them.